golden_response/staging/get_pop_obs/moscow.json

```diff
@@ -8942,6 +8942,18 @@
             "Count_Person_PerArea"
           ]
         },
+        {
+          "id": "dc/o/9egf628qj990b",
+          "type": "Observation",
+          "measured_prop": "count",
+          "measured_value": 11979529,
+          "measurement_method": "GoogleKGHumanCurated",
+          "provenance_id": "dc/97m6fj1",
+          "observation_date": "2013",
+          "stat_var_ids": [
+            "Count_Person"
+          ]
+        },
```

Comment:
I spent half an hour trying to debug this diff. Seemingly, the content of the Obs stayed the same, and yet the DCID changed.

My best guess is that @shifucun manually edited the resolved MCF to add "measurementMethod=GoogleKGHumanCurated". So really, the obs DCID on the LHS is bogus and the RHS is correct (unit test illustrating this: cl/328421816).

For future reference, we should always edit unresolved MCFs and re-resolve.